examples/basic/package.json

```diff
@@ -3,6 +3,11 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "react": "../../node_modules/react",
+    "react-dom": "../../node_modules/react-dom/profiling",
+    "react-final-table": "../../dist"
```

Comment:
So here I wonder if it'd be better to use `yarn link` or `npm link` instead of pointing to dist, what do you think?

Reply:
So link would still require the dist folder but would allow it to work with the latest published version out of the box. I'm okay if that's what you would like.

Reply:
Yeah, I think that would be better.  Generally it's how I've seen others do it and I think the dev cycle may be slightly better that way.